### ISSUE TITLE
ohai root_group leads to invalid byte sequence in US-ASCII on Russian version of Windows 2016

### ISSUE THREAD

**Issue by jugatsu** [chef/ohai#1233]
## Description

`chef-client::config` recipe fails with the following error:

```
Compiling Cookbooks...
       
       ================================================================================
       Recipe Compile Error in C:/Users/vagrant/AppData/Local/Temp/kitchen/cache/cookbooks/gf_chef_client/recipes/default.rb
       ================================================================================
       
       ArgumentError
       -------------
       invalid byte sequence in US-ASCII
       
       System Info:
       ------------
       chef_version=13.10.4
       platform=windows
       platform_version=10.0.14393
       ruby=ruby 2.4.4p296 (2018-03-28 revision 63013) [x64-mingw32]
       program_name=C:/opscode/chef/bin/chef-client
       executable=C:/opscode/chef/bin/chef-client
```

`node[root_group]` returns non correct output so it fails https://github.com/chef-cookbooks/chef-client/blob/master/libraries/helpers.rb#L53

![2018-08-18 13 57 18](https://user-images.githubusercontent.com/2243323/44298676-e81e8d80-a2ef-11e8-9ff1-daf4f8eaf5b0.png)

## Ohai Version
Ohai: 13.10.0

## Platform Version

Windows Server 2016 Datacenter Evaluation Russian Edition

```powershell
PS C:\Users\vagrant> get-winsystemlocale

LCID             Name             DisplayName
----             ----             -----------
1049             ru-RU            Русский (Россия)
```

## Ohai Output

[2018-08-18T04:00:06-07:00] DEBUG: Loading plugin at C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/ohai-13.10.0/lib/ohai/plugins/windows/virtualization.rb
[2018-08-18T04:00:06-07:00] DEBUG: Loading plugin at C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/ohai-13.10.0/lib/ohai/plugins/zpools.rb
[2018-08-18T04:00:06-07:00] DEBUG: Plugin RootGroup took 0.0 seconds to run.
[
  "��������������"
]
[2018-08-18T04:00:06-07:00] DEBUG: Ohai took 0.34400000000000014 total seconds to run.

**Comment by jugatsu** [comment]
![2018-08-19 11 00 25](https://user-images.githubusercontent.com/2243323/44306823-29ba4180-a39f-11e8-96a3-625abe00f3ea.png)

**Comment by jugatsu** [comment]
![2018-08-19 12 44 29](https://user-images.githubusercontent.com/2243323/44307612-dbf90580-a3ad-11e8-8803-920ed7e5acd6.png)

**Comment by jugatsu** [comment]
Can reproduce on chef 14
```cmd
C:\Users\vagrant>ohai -v
Ohai: 14.3.1

C:\Users\vagrant>chef-shell
loading configuration: none (standalone session)
Session type: standalone
Loading................done.

This is the chef-shell.
 Chef Version: 14.3.37
 https://www.chef.io/
 https://docs.chef.io/

run `help' for help, `exit' or ^D to quit.

Ohai2u vagrant@vagrant-2016!
chef (14.3.37)> node['root_group']
 => "\xC0\xE4\xEC\xE8\xED\xE8\xF1\xF2\xF0\xE0\xF2\xEE\xF0\xFB"
```

**Comment by jugatsu** [comment]
Closed by #1240
